src/Neo/SmartContract/Manifest/ContractMethodDescriptor.cs

```diff
@@ -28,6 +28,11 @@ public class ContractMethodDescriptor : ContractEventDescriptor, IEquatable<Cont
         /// </summary>
         public ContractParameterType ReturnType { get; set; }
 
+        /// <summary>
+        /// NEP-25 extended return type
+        /// </summary>
+        public ExtendedType ExtendedReturnType { get; set; }
```

Comment:
i think nep 25 is for all compound valud, not just return type? or this pr only focus on return type?